x-pack/plugins/secops/public/components/timeline/properties/styles.tsx

```diff
@@ -74,14 +73,12 @@ export const StyledStar = styled(EuiIcon)`
   cursor: pointer;
 `;
 
-export const PropertiesLeft = styled.div`
-  align-items: center;
-  display: flex;
-  flex-direction: row;
+export const PropertiesLeft = styled(EuiFlexGroup)`
+  width: 100%;
```

Comment:
Removing this in favor of EuiFlexGroup is going to help us have a chance at fixing IE-11. Much appreciated 🥇